.github/workflows/cypress.yml

```diff
@@ -79,3 +79,96 @@ jobs:
         with:
           name: cypress-screenshot-artifacts
           path: cypress/screenshots
+
+      - name: Archive Cypress Mochawesome test results
+        uses: actions/upload-artifact@v2
+        if: ${{ always() }}
+        with:
+          name: cypress-mochawesome-test-results-artifact-${{ matrix.ci_node_index }}
+          path: cypress/results
+
+  mochawesome-report:
+    if: ${{ always() }}
+    needs: build
+    name: Generate Mochawesome Report
+    runs-on: ubuntu-latest
+    defaults:
+      run:
+        working-directory: testing-tools-team-dashboard-data
+    steps:
+      - name: Configure AWS Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: "us-gov-west-1"
+      
+      - name: Set BIGQUERY_SERVICE_CREDENTIALS Environment Variable
+        uses: "marvinpinto/action-inject-ssm-secrets@v1.1.1"
```

Comment:
Should this be `@latest` like in line #113?

Reply:
Thanks. I updated it to `@latest`.